tools/wptrunner/wptrunner/browsers/chrome_android.py

```diff
@@ -189,15 +189,21 @@ def logcat_cmd(self):
         cmd.extend(['logcat', '*:D'])
         return cmd
 
-    def maybe_parse_tombstone(self, logger):
-        if self.stackparser_script:
-            cmd = [self.stackparser_script, "-a", "-w"]
+    def check_crash(self, process, test):
+        self.maybe_parse_tombstone()
+        # Existance of a tombstone does not necesssary mean test target has
```

Comment:
nit: couple typos
Existence, and
necessarily

Reply:
done